graphql-schema/proposal.graphql

```diff
@@ -66,13 +66,27 @@ type ProposalTallyResult {
   outstandingOption: ProposalVoteOption
 }
 
+"""
+Filter by address's role in proposals
+or operator will be used when more than one is* field is specified
+"""
+input ProposalAddressFilter {
+  address: String!
+  isDepositor: Boolean!
+  isVoter: Boolean!
+  isSubmitter: Boolean!
+}
```

Comment:
Does that mean no filtering if we provide an account but with all the is* field set to false?

Reply:
I think this would be an edge case and we need to define what the default behavior would be if all is* is set false.

- one option would be no filter applied as you mentioned.
- another option could be no results are returned (I was planning to implement this)